internal/host/statemachine.go

```diff
@@ -769,6 +770,62 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th TransitionHandler) stat
 		},
 	})
 
+	// Time out while host installationInProgress
```

Comment:
These commends are all the same, can you replace them with something explaining the difference between these 4 transitions?

Reply:
OK